### ISSUE TITLE
Serializing a language with an annotation that doesn't say what it annotates breaks

### ISSUE THREAD

**Issue by dslmeinte** [LionWeb-io/lionweb-typescript#177]
If `Annotation.annotates` is `null` (meaning that the reference is not set), the serialization throws (an unhelpful exception). (The computed name should just be left `undefined`.)

**Comment by dslmeinte** [comment]
See also: https://github.com/LionWeb-io/specification/issues/307

**Comment by enikao** [comment]
This would be an invalid language, so it is invalid according to the relevant document (-:

**Comment by dslmeinte** [comment]
Yeah, but the serialization should preferably not just crash on invalid languages.

**Comment by dslmeinte** [comment]
Unit test added to PR #198
